ozaria/site/components/common/LayoutChrome.vue

```diff
@@ -42,7 +123,7 @@
   </div>
 </template>
 
-<style lang="sass" scoped>
+<style lang="sass">
```

Comment:
What CSS is global here?  We should always use scoped here to prevent conflicts in the future.  If some set of styles is global it should be loaded with some global style sheet.

Reply:
This is not scoped in order to pick up the tooltips created by `v-tooltip`.
These tooltips as far as I know are local to this system, however I'm unsure of how to select them when scoped is active.

Reply:
Do you mean that its not scoped so that the CSS will apply to the tooltip?  If so try prefixing line 269 with `/deep/ `

Reply:
May be another line, basically `/deep/` makes the styles apply in nested scenarios like this.

Reply:
`/deep/` didn't work as the tooltip gets applied to the end of the body and thus isn't nested. In this case I [mixed local and global styles](https://vue-loader.vuejs.org/guide/scoped-css.html#mixing-local-and-global-styles) to have the vast majority of styles scoped.
Only the tooltip selector ends up behaving globally.